packages/components/src/List/List.scss

```diff
@@ -24,5 +24,14 @@
 		flex-grow: 1;
 		flex-shrink: 1;
 		box-shadow: $shadow-default;
+		height: 100%;
+	}
+
+	:global(.tc-list-progress) {
```

Comment:
This should not be defined here. It is something in VirtualizedList, put it in its style please.
Plus, we can use css modules instead of global class

Reply:
Which way is better from your point of you, add as css module?

Reply:
add as css module in vrituazedList.scss